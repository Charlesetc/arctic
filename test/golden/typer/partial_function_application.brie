define main [ [
          define add : a b [ 2 ]
                  add 2
] ]
