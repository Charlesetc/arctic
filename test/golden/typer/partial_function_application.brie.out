{[6]=>Literal(subtype=integer), [5, 7]=>Literal(subtype=integer), [10, 8]=>Function_literal(takes=3, returns=7), [0, 1, 4, 9, 11]=>Function_literal(takes=2, returns=10)}